Gitweb:     http://git.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commit;h=fb49161027e1938c34fc97d1136735e1d4209df6
Commit:     fb49161027e1938c34fc97d1136735e1d4209df6
Parent:     653e91d01fa4d39d2ed06a8c2096fef08b00ee7e
Author:     Dmitry Torokhov <dmitry.torokhov@gmail.com>
AuthorDate: Thu Jan 17 12:01:58 2008 -0500
Committer:  Dmitry Torokhov <dmitry.torokhov@gmail.com>
CommitDate: Thu Jan 17 12:01:58 2008 -0500

    Input: ALPS - fix sync loss on Acer Aspire 5720ZG
    
    The recently added support for Dell Volstro 1400 was causing protocol
    synchronization errors on Acer Aspire 5720ZG, fix it.
    
    Signed-off-by: Dmitry Torokhov <dtor@mail.ru>
---
 drivers/input/mouse/alps.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/input/mouse/alps.c b/drivers/input/mouse/alps.c
index 2b5ed11..b346a3b 100644
--- a/drivers/input/mouse/alps.c
+++ b/drivers/input/mouse/alps.c
@@ -54,7 +54,7 @@ static const struct alps_model_info alps_model_data[] = {
 	{ { 0x20, 0x02, 0x0e },	0xf8, 0xf8, ALPS_PASS | ALPS_DUALPOINT }, /* XXX */
 	{ { 0x22, 0x02, 0x0a },	0xf8, 0xf8, ALPS_PASS | ALPS_DUALPOINT },
 	{ { 0x22, 0x02, 0x14 }, 0xff, 0xff, ALPS_PASS | ALPS_DUALPOINT }, /* Dell Latitude D600 */
-	{ { 0x73, 0x02, 0x50 }, 0xcf, 0xff, ALPS_FW_BK_1 } /* Dell Vostro 1400 */
+	{ { 0x73, 0x02, 0x50 }, 0xcf, 0xcf, ALPS_FW_BK_1 } /* Dell Vostro 1400 */
 };
 
 /*
